Gemfile

```diff
@@ -54,6 +54,3 @@ group :development do
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
-
-# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
```

Comment:
Okay. There's no harm for our purposes in removing this, but there's also no harm leaving it there.